source/MaterialXTest/GenShaderUtil.cpp

```diff
@@ -421,6 +395,65 @@ void ShaderGeneratorTester::addSkipNodeDefs()
 {
 }
 
+
```

Comment:
@niklasharrysson : I am keeping these here since the intent was always to derive render tests from gen tests so don't want to put in "fake" code. I found making up constant ids was more work as just keeping the existing code.